### PR TITLE
Portfolio: Cards and about me section pull request

### PR DESCRIPTION
**Pull request by adamteddychang**
In this branch, the added features are:

- 2 types of Project cards
- About me Section
- background-image of the about me section
- Tech stack section

### CODE REVIEW

**Review by wandji20 — CHANGES_REQUESTED** [review]
Hello, @adamteddychang 
-----
Great :muscle:  job :raised_hands:  implementing most of the requirements for your portfolio project.
However, there are a few issues you need to address to have this project approved.
## To Highlight! :clap:  :green_circle: 

:heavy_check_mark:  You did well following the Figma design :+1: 
:heavy_check_mark:  Your Pull request description and commit messages are descriptive.

## Status: Require Changes :recycle: :red_circle: 
Please check the comments below and under the review and make all required changes.
Optional comments are not required to mark this project approved

- [ ] It will be nice to proofread your README and remove all unnecessary lines.

**Happy coding :computer:** 
**_Do not hesitate to make comments in this PR in case of doubts._**
**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 1]
```diff
@@ -23,18 +23,128 @@ <h2 id="second-title">Welcome to my Portfolio page.</h2>
...
+        <h1 class="bk-img-title">Progessional Art Printing Data</h1>
+        <p class="bk-img-p">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+        <ul class="lang-contain_2">
+            <li class="lang-li_2">html</li>
+            <li class="lang-li_2">bootstrap</li>
+            <li class="lang-li_2">Ruby</li>
+        </ul>
+        <button class="st-see-btn_2" type="button">See Project</button>
+
+    </div>
```
[the remark this answers is not in the document]
Comment (adamteddychang) [reply index.html:108]: All the projects are wrapped in `<section id="works-container">`, is that not enough? Please let me know how I can approach this, thank you.

File: index.html [hunk 2]
```diff
@@ -23,18 +23,128 @@ <h2 id="second-title">Welcome to my Portfolio page.</h2>
...
+    <div class="bk-img-proj">
+        <h1 class="bk-img-title">Progessional Art Printing Data</h1>
+        <p class="bk-img-p">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+        <ul class="lang-contain_2">
+            <li class="lang-li_2">html</li>
+            <li class="lang-li_2">bootstrap</li>
+            <li class="lang-li_2">Ruby</li>
+        </ul>
+        <button class="st-see-btn_2" type="button">See Project</button>
```
[the remark this answers is not in the document]
Comment (adamteddychang) [reply index.html:62]: Added buttons animation.

File: index.html [hunk 3]
```diff
@@ -23,18 +23,128 @@ <h2 id="second-title">Welcome to my Portfolio page.</h2>
...
+
+<section class="about_me">
+    <h2 class="about_me_title">About me</h2>
+    <div class="bk_img_d"> 
+<div class="about_me_d">
+        <p class="about_me_p">Hello I’m a software developer! I can help you build a product, 
+    feature or website 
+    Look through some of my work and experience! 
+    If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+        <button class="st-see-btn_3" type="button">Get My Resume</button>
```
[the remark this answers is not in the document]
Comment (adamteddychang) [reply index.html:119]: Added button styling.

File: index.html [hunk 4]
```diff
@@ -23,18 +23,128 @@ <h2 id="second-title">Welcome to my Portfolio page.</h2>
...
+        <li class="tech_stack">RSPec</li>
+        <li class="tech_stack">CapyBara</li>
+        <li class="tech_stack">Selenium</li>
+        <li class="tech-title_3">Skills</li>
+        <li class="tech_stack">Codekit</li>
+        <li class="tech_stack">Github</li>
+        <li class="tech_stack">Codepen</li>
+        <li class="tech_stack">Gitlab</li>
+        <li class="tech_stack">Terminal</li>
+    </ul>
```
[the remark this answers is not in the document]
Comment (adamteddychang) [reply index.html:142]: Completed. I have created more `ul` to put the items in it.

File: index.html [hunk 5]
```diff
@@ -23,18 +23,128 @@ <h2 id="second-title">Welcome to my Portfolio page.</h2>
...
+    
+    <ul></ul>
+</div>
+
+</section>
```
[the remark this answers is not in the document]
Comment (adamteddychang) [reply index.html:147]: Added flexbox properties to this section.

File: index.html [hunk 6]
```diff
@@ -23,18 +23,128 @@ <h2 id="second-title">Welcome to my Portfolio page.</h2>
 <p id="description">I am a Software Developer in training</p>
 <p id="main-p">I can build basic portfolio page, landing page, or website for your business. 
 If you are interested, lets hang out! My socials are down there!</p>
-<img src="Images/Illustration.png" alt="background-img" id="back-img">
+<img src="Images/Illustration.png" alt="background-img" class="back-img">
```
[the remark this answers is not in the document]
Comment (adamteddychang) [reply index.html]: I originally had this image in css, but it was the request of the first code reviewer of the last project that asked me to move it to HTML, that's why it is there. I have added it back to css.

File: index.html (hunk 4 above)
[the remark this answers is not in the document]
Comment (adamteddychang) [reply index.html]: By doing this I am unable to past the linter check, it says element `ul` not allowed as child of element `ul`.

[the remark this answers is not in the document]
Comment (adamteddychang) [reply index.html]: My bad, my coding partner and I figured it out. It works

File: index.html [hunk 7]
```diff
@@ -23,18 +23,128 @@ <h2 id="second-title">Welcome to my Portfolio page.</h2>
...
+        </ul>
+        <button id="st-see-btn" type="button">See Project</button>
+    </div>
+    <div class="bk-img-proj">
+        <h1 class="bk-img-title">Progessional Art Printing Data</h1>
+        <p class="bk-img-p">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+        <ul class="lang-contain_2">
+            <li class="lang-li_2">html</li>
+            <li class="lang-li_2">bootstrap</li>
+            <li class="lang-li_2">Ruby</li>
```
[the remark this answers is not in the document]
Comment (adamteddychang) [reply index.html:59]: I have added some styling to those blocks.

**Comment by adamteddychang** [comment]
I have fixed the contents upon request, please let me know if there are other necessary changes, thank you.

File: index.html [hunk 8]
```diff
@@ -23,18 +23,137 @@ <h2 id="second-title">Welcome to my Portfolio page.</h2>
...
+    <li class="social_icons"><a href="#"><img src="Images/Medium.png" alt="medium logo" height="20" class="icons"></a></li>
 </ul>
 
 </div>
 </section>
+<section id="works-container">
+    <h1 class="works-title">My Recent Works</h1>
+    <div class="projects-container">
+        <img src="Images/yoga.png" alt="yoga png" height="237" class="projects-img">
+        <h1 class="projects-title">Multi-Post Stories</h1>
```
Comment (DanSam5K) [inline index.html:41]: - [Optional] It would be nice to limit the number of h1 tags in a webpage to just one use other like h2 instead 😉

**Review by DanSam5K — APPROVED** [review]
## Hi @adamteddychang , ✌️ Approved 🎉 
You nailed 🔨  this project real good and was amazing implementing the changes suggested by the previous reviewer 👏 👏 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 🎉 

- [ ] [Optional] It is a good convention to limit the use of h1 you may consider changing those h1 tag in your project to use may be h2, what do you think? 🤔 

### Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**